docs/installation.rst

```diff
@@ -9,16 +13,252 @@ Fantastic! Thank you!
 
-* a NodeJS-based web server
-* a PostgresQL database
-* a Redis server to store user sessions
-* an ElasticSearch server to handle search
+Cloning
+*******
 
-The standard way to run these dependencies is using Docker.
-For a primer on what Docker is and how to use it, visit `DockerGettingStarted`_.
\ No newline at end of file
+To clone the repository and point the local HEAD to the latest commit in the stable branch, something like the following command should work
+
+::
+    
+    git clone --recurse-submodules https://github.com/metabrainz/bookbrainz-site.git
+
+Since this project makes use of git submodules, you need to use ``git clone --recurse-submodules`` to clone it.
+
+Alternatively, to manually initialize submodules, run these two commands
+::
+
+    git submodule init
+    git submodule update
+
+Currently we are using these submodules
+
+* `MonkeyDo/Lobes`_ in ``src/client/stylesheets/lobes``
+
+Configuration
+*************
+
+Create a copy of ``config/config.json.example`` and rename it to ``config.json``. You can do this by running this command from the bookbrainz-site directory.
+
+::
+
+    cp config/config.json.example config/config.json
+
+If you want to be able to sign-up and edit, you will need to set up authentication under ``musicbrainz``.
+To get the ``clientID`` and ``clientSecret`` tokens, head to `MusicBrainz`_ and register a new developer application.
+
+Make sure to enter the callback URL as ``http://localhost:<port>/cb`` (port: 9099 by default).
+
+You can then copy the tokens for that developer application and paste as strings in your ``config/config.json``. The tokens and callback URL in your ``config/config.json`` needs to match exactly the one for the developer application.
+
+Docker Installation
+===================
+The easiest way to get a local developement server running is using Docker. This will save you a fair amount of set up.
+
+You'll need to install Docker and Docker-compose on your development machine:
+
+* `Docker`_
+*  `Docker-compose`_
+
+When it is installed, follow the below instructions step by step.
+
+.. note:: 
+  If you are using docker-toolbox you need to replace `elasticsearch:9200 <https://github.com/metabrainz/bookbrainz-site/blob/master/config/config.json.example#L30>`_ with ip address of your docker-machine in config.json.
+  To get ip address of your docker machine use command ``docker-machine ip default``
+
+Database set-up
+***************
+When you first start working with BookBrainz, you will need to perform some initialization for PostgreSQL and import the latest BookBrainz database dump.
+
+Luckily, we have a script that does just that: from the command line, in the ``bookbrainz-site`` folder, type and run ``./scripts/database-init-docker.sh``. The process may take a while as Docker downloads and builds the images. Let that run until the command returns.
+
+The latest database dump can be found `here <http://ftp.musicbrainz.org/pub/musicbrainz/bookbrainz/latest.sql.bz2>`_
```

Comment:
I would make this a `..note::` since they likely won't need to download it themselves (the database-init-docker script should take care of that)